src/studio/src/designer/frontend/packages/schema-editor/src/components/SchemaInspector.module.css

```diff
@@ -25,6 +25,7 @@
   border-bottom: 1px solid #c9c9c9;
   box-shadow: none !important;
   color: #000;
+  background-color: transparent !important;
```

Comment:
Is this because of hard-set styles within MatUi?

Reply:
Yes, I guess so. MUI is enforcing it own styles. So this was a solution on that.